NOTES.WIN

```diff
@@ -83,6 +83,16 @@
  is, of course, to choose a different set of directories by using
  --prefix and --openssldir when configuring.
 
+
+ Special notes for Universal Windows Platform builds, a.k.a. VC-*-UWP
+ --------------------------------------------------------------------
+
+ - UWP targets only support building the static and dynamic libraries.
+
+ - You should define the platform type to "uwp" and the target arch via
+   "vcvarsall.bat" before you compile. For example, if you want to build
+   "arm64" builds, you should type "vcvarsall.bat x86_arm64 uwp".
```

Comment:
So, sorry for going around in loops on this one, but as I understand it the "uplink" disable in the Configuration isn't working...so now we need to add back in the statement about needing to add no-uplink on the command line

Reply:
I will add it back to note.